.github/ISSUE_TEMPLATE/initiate_spiral.yaml

```diff
@@ -0,0 +1,124 @@
+name: Problem Statement
+description: Please provide a statement of the problem to be considered for research.
+labels: ['Step 1 - Initiation']
+body:
+  - type: markdown
+    attributes:
+      value: A spiral must be in support of an approved research effort.  If a research effort has not started, you should start by creating a [problem statement issue](https://github.com/usnistgov/OSCAL-DEFINE/issues/new?assignees=&labels=Step+1+-+Initiation&projects=&template=problem_statement.yaml).
+
+
+
+  - type: input
+    id: github-research-effort
+    attributes:
+      label: OSCAL DEFINE Research Effort Link
+      description: Provide a link to the associated research effort issue.
+      placeholder: ""
+    validations:
+      required: true
+
+  - type: input
+    id: github-project
+    attributes:
+      label: GitHub Project Link
+      description: Typically, a spiral relates to an existing project.  OSCAL is the default below.
```

Comment:
Suggestion:
description: Typically, a spiral relates to an existing project. The default project is OSCAL (listed below). Please provide below the URL of the targeted project if different than core OSCAL.